### ISSUE TITLE
Data preparation

### ISSUE THREAD

**Issue by Lichuchen** [liuquande/SRC-MT#1]
Hi~great work, thanks for the code.

In readme, the dataset organization is not specifically described. 
For example, the ISIC 2018 data. train.csv, val.csv and test.csv are not provided in the official website, it would be very helpful for you to tell me how to generate the csv file or could you upload these files in this repo?

Thanks a lot

**Comment by liuquande** [comment]
Hi Lichuchen,

Thanks for your interest on our work.

There is a official groundtruth file called "ISIC2018_Task3_Training_GroundTruth.csv" in the training  data of Task 3 in ISIC 2018. 
This csv file contains the groundtruth labels of all training data.
You can split all the rows in this csv file into 7:1:2 for training, validation and testing, which can directly serve as the train.csv, val.csv and test.csv in our code.
